openmmtools/testsystems.py

```diff
@@ -350,6 +357,19 @@ def positions(self):
         del self._positions
 
     @property
+    def topology(self):
```

Comment:
So with the deepcopy here, it's essentially impossible to edit the underlying topology via `testsystem.topology`.  I guess this is probably a good idea here, but it's worth thinking about.